File: IDSelector/app/src/main/java/hu/bme/idselector/data/Person.kt
package hu.bme.idselector.data

data class Person(
    val name: String?,
    val sex: String?,
    val nationality: String?,
    val dateOfBirth: String?,
    val dateOfExpiry: String?,
    val documentNr: String?,
    val can: String?,
    val placeOfBirth: String?,
    val nameAtBirth: String? = null,
    val mothersName: String?,
    val authority: String?,
)

File: ClipboardExtension/person.js
export class Person {
  constructor(data) {
    console.log(data);
    if (!data) {
      this.name = null;
      this.document_nr = null;
      this.date_of_birth = null;
      return;
    }
    this.name = data.name;
    this.document_nr = data.documentNr;
    this.date_of_birth = data.dateOfBirth;
  }

  setData(data) {
    this.name = data.name;
    this.document_nr = data.documentNr;
    this.date_of_birth = data.dateOfBirth;
  }
}

const person = new Person();

export { person };